scylla/src/transport/locator/mod.rs

```diff
@@ -21,6 +21,23 @@ use std::{
 };
 use tracing::debug;
 
+/// TODO(michael): Docs
+#[derive(Debug, Default, Clone)]
+pub struct ReplicationConfigs {
+    // Keyspaces to exclude from precomuted replica sets
+    keyspaces_to_exclude: Vec<String>,
+}
+
+impl ReplicationConfigs {
+    pub fn exclude_keyspace(&mut self, keyspace: String) {
+        self.keyspaces_to_exclude.push(keyspace)
+    }
+
+    pub fn keyspaces_to_exclude(&self) -> &Vec<String> {
+        &self.keyspaces_to_exclude
+    }
+}
+
```

Comment:
Let's create a `ReplicasPrecomputationList` enum:
```rust
enum ReplicasPrecomputationList {
    Allowlist { allowed_keyspaces: Vec<String> },
    Denylist { disallowed_keyspaces: Vec<String> },
}
```
and store it `ReplicationConfig` as `Option<ReplicaPrecomputationList>`.